commands/admin/copy_7tv_emote.py

```
@@ -0,0 +1,306 @@
+"""7TV Emote Copy Command — browse and add a streamer's 7TV emotes to the server.
+
+Provides a paginated UI (like Brawl Stars brawlers) showing the emotes from a
+streamer's 7TV emote set, with a button to copy each emote to the server.
+"""
+
+from __future__ import annotations
+
+import logging
+
+import aiohttp
```

Flake8 (7.3.0)

[error] 'aiohttp' imported but unused

(F401)